lib/custom_errors.rb

```diff
@@ -4,7 +4,7 @@ def self.included(base)
     base.rescue_from Exception, with: ->(exception) { render_error 500, exception }
 
     base.rescue_from ActionController::RoutingError,
-                     ActionController::UnknownController,
+                     ActionController::ActionControllerError,
```

Comment:
Here I have used the parent class `ActionControllerError` instead of `UnknownController`.